app/Features/QuerySpecificAnimeSeasonHandler.php

```diff
@@ -26,8 +27,9 @@ protected function getSeasonItems($request, ?AnimeTypeEnum $type): Builder
          */
 
         [$from, $to] = $this->getSeasonRange($request->year, $request->season);
+        $premiered = ucfirst($request->season)." {$request->year}";
 
-        return $this->repository->getAiredBetween($from, $to, $type);
+        return $this->repository->getAiredBetween($from, $to, $type, $premiered);
```

Comment:
This is ok for now. I'm going to create an issue to refactor this bit, because the `getSeasonRange` should be inside the `DefaultAnimeRepository` class, and the signature of the `getAiredBetween` should be `getAiredBetween($year, $season, $type)` or we should create a `getBySeason($year, $season, $type)` function.

Reply:
https://github.com/jikan-me/jikan-rest/issues/449